mne/io/kit/constants.py

```diff
@@ -71,6 +71,13 @@
 KIT_NY.LPFS = [10, 20, 50, 100, 200, 500, 1000, 2000]
 
 
+# Maryland-system channel information
+# Virtually the same as the NY-system except new ADC circa July 2014
+# 16-bit A-to-D converter, one bit for signed integer. range +/- 32,768
+KIT_MD = Bunch(**KIT_NY)
+KIT_MD.DYNAMIC_RANGE = 2 ** 16 / 2
```

Comment:
I was just making it explicit that it is 16-bit ADC and that one bit was being removed for the sign. The comment above is good enough